src/services/housing.js

```diff
@@ -189,42 +167,162 @@ const changeApartmentAppEditor = async (applicationID, newEditorUsername) => {
  * @return {Promise.<ApplicationDetails>} Application details
  */
 const getApartmentApplication = async (applicationID) => {
-  try {
-    return await http.get(`housing/apartment/applications/${applicationID}/`);
-  } catch (err) {
-    if (err?.status === 404 || err?.name?.includes('NotFound')) {
-      console.log(
-        'Received 404 indicates that the requested application was not found in the database',
-      );
-    } else {
-      throw err;
-    }
-    return null;
-  }
```

Comment:
Things like this indicate there are some pretty big problems with the recent merge, as it is trying to remove things that you did not even modify. This merge mistake will have to be fixed before I can continue the review, since it is very difficult to tell which things are meant to be here. Also merging it in this state would likely break most of ApartApp